### ISSUE TITLE
Binary '.exe' is not defined

### ISSUE THREAD

**Issue by itmuckel** [nim-lang/nimble#735]
When using `nimble run` on Windows I get
```
Error: Binary '.exe' is not defined in 'mypackage' package.
```

How to reproduce:
* Use nimble init
* Choose Binary as project type
* Run `nimble build`
* Run `nimble run`

Nim version 1.0.2
Nimble version 0.11.0
(Both installed with choosenim -> stable)

**Comment by itmuckel** [comment]
Turns out that  `nimble run` needs the name of the binary it should run. So `nimble run mypackage` actually runs the executable.

As a feature request it would be nice, if `nimble run` would run the executable name specified in `bin` in the nimble file by default.

**Comment by genotrance** [comment]
Nimble supports multiple bins so it might be required to clarify which one you want to run. I guess it could be inferred if there is only one.

**Comment by rishavs** [comment]
Can we change this error message. It was fairly confusing to me as a new user. 

Perhaps a hint line;
"Note: You need to the package name to "nimble run <package name>" command so that nimble knows which executable to run."
would be very helpful.

**Comment by inv2004** [comment]
https://github.com/nim-lang/nimble/pull/761

**Comment by dom96** [comment]
Fixed via #761 I assume.
